dd-sdk-android-core/src/test/kotlin/com/datadog/android/utils/forge/Configurator.kt

```diff
@@ -43,6 +43,27 @@ internal class Configurator :
         forge.addFactory(ProcessInfoForgeryFactory())
         forge.addFactory(DeviceInfoForgeryFactory())
         forge.addFactory(DatadogContextForgeryFactory())
+        forge.addFactory(DataUploadConfigurationForgeryFactory())
+
+        // UploadStatus
+        forge.addFactory(HttpRedirectStatusForgeryFactory())
+        forge.addFactory(HttpClientRateLimitingStatusForgeryFactory())
+        forge.addFactory(HttpClientErrorForgeryFactory())
+        forge.addFactory(HttpServerErrorForgeryFactory())
+        forge.addFactory(InvalidTokenErrorStatusForgeryFactory())
+        forge.addFactory(NetworkErrorStatusForgeryFactory())
+        forge.addFactory(RequestCreationErrorStatusForgeryFactory())
+        forge.addFactory(SuccessStatusForgeryFactory())
+        forge.addFactory(UnknownErrorStatusForgeryFactory())
+        forge.addFactory(UnknownStatusForgeryFactory())
+
+        // RemovalReason
+        forge.addFactory(RemovalReasonFlushedForgeryFactory())
+        forge.addFactory(RemovalReasonPurgedForgeryFactory())
+        forge.addFactory(RemovalReasonInvalidForgeryFactory())
+        forge.addFactory(RemovalReasonObsoleteForgeryFactory())
+        forge.addFactory(RemovalReasonIntakeCodeForgeryFactory())
+        forge.addFactory(RemovalReasonForgeryFactory())
```

Comment:
same, cannot it be in a single `RemoveReason` factory or we need such separation?

Reply:
Same as above